digdag-docs/src/operators/require.md

```diff
@@ -60,6 +60,17 @@
   project_id: 12345
   ```
 
+* **project_name**: project_name
+
+  Name of another project. You can kick another project's workflow by setting this parameter.
+
+  Examples:
+
+  ```
+  require>: another_project_wf
+  project_name: another_project
+  ```
+
```

Comment:
Maybe it's worth explaining what will happen if the target project doesn't exist.

Reply:
Got it. Will add.

Reply:
I updated the document.